pyfaidx/__init__.py

```diff
@@ -293,7 +293,7 @@ def __init__(self, filename, default_seq=None, key_function=lambda x: x.split()[
         self.filename = filename
 
         filenameLower = filename.lower()
-        if filenameLower.endswith('.bgz'):
+        if filenameLower.endswith('.bgz') or filenameLower.endswith('.gz'):
```

Comment:
Sure. I'm a bit old fashioned, and don't like to potentially call the same function multiple times. If I was being even more efficient I'd have done something like `endswith = filename.lower().endswith` but that also creates a local variable. What don't you like about creating locals?  In any case, I'll change it.